app/src/components/course/CourseStatusBadge.jsx

```diff
@@ -1,23 +1,50 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { getRegistrationByStaffAndCourseId } from "src/api/registration";
 
-export default function CourseStatusBadge({registrationStatus, completionStatus}) {
+export default function CourseStatusBadge({ staffId, courseId }) {
   const statusToBadgeClass = {
-    "Completed": "bg-green-100 text-green-800",
-    "Rejected": "bg-red-100 text-red-800", 
-    "Waitlisted": "bg-blue-200 text-blue-800",
-    "Registered": "bg-yellow-100 text-yellow-800",
-    "Not Registered": "bg-purple-100 text-purple-800"
+    Completed: "bg-green-100 text-green-800",
```

Comment:
rmb the inverted commas around your keys

Reply:
Oops, will make changes thanks!